src/styles/globals.scss

```diff
@@ -299,3 +299,27 @@
     margin-bottom: 0 !important;
   }
 }
+
+.IntroCards {
+  display: grid;
+  gap: pxToRem(12px);
+  grid-template-columns: 1fr 1fr;
+
+  .Card {
+    display: flex;
+    flex-direction: column;
+    gap: pxToRem(8px);
+    justify-content: space-between;
+
+    :nth-child(2) {
+      flex: 1;
+    }
+  }
+}
+
+// =============================================================================
+// Margin | Padding
+// =============================================================================
+.padding-top-30 {
```

Comment:
do you have any preferences on defining classes? @quietbits

Reply:
Let's use `gap` for spacing as much as possible. That will give us more flexibility with layouts.

Reply:
`gap` didn't work for me since it only needed an extra `padding-top` for the main cta button